nitrorole/nitrorole.py

```diff
@@ -317,6 +317,36 @@ async def cog_disabled_in_guild(self, guild: Optional[discord.Guild]) -> bool:
             return False
         return await func(self, guild)
 
+    @nitrorole.command(name="settings", aliases=["show", "showsettings", "setting"])
+    async def nitrorole_settings(self, ctx: GuildContext) -> None:
+        """
+        Shows the current NitroRole settings for the server.
+        """
+        
+        guild = ctx.guild
+        guild_data = await self.get_guild_data(guild)
+        role_id = guild_data.role_id
+        channel_id = guild_data.channel_id
+        unassign_on_boost_end = guild_data.unassign_on_boost_end
+
+        # Check if the channel is set, implying the cog is enabled
+        is_enabled = "Enabled" if channel_id else "Disabled"
+
+        role = guild.get_role(role_id) if role_id else None
+        role_mention = role.mention if role is not None else "None"
+        channel = channel_id and guild.get_channel(channel_id) if channel_id else None
+        channel_mention = channel.mention if channel is not None else "None"
+        unassign_status = "Yes" if unassign_on_boost_end else "No"
+
+        await ctx.send(
+            "## __**NitroRole's Guild Settings**__\n\n"
+            ">>> "
+            f"- **Auto-assign role:** {role_mention}\n"
+            f"- **Boost message channel:** {channel_mention}\n"
+            f"- **Unassign role on boost end:** {unassign_status}\n"
+            f"- **Cog status:** {is_enabled}"
```

Comment:
Move this to top and rename to "Enabled"
```suggestion
            f"- **Enabled:** {is_enabled}"
```